doc/design.rst

```diff
@@ -298,8 +300,8 @@ example, the Pack ``73d04e61`` contains two data Blobs and one Tree
 blob, the plaintext hashes are listed afterwards. The ``length`` field
 corresponds to ``Length(encrypted_blob)`` in the pack file header.
 Field ``uncompressed_length`` is only present for compressed blobs and
-therefore is never present in version 1. It is set to the value of
-``Length(blob)``.
+therefore is never present in the Repository format version 1. It is set
```

Comment:
Total nitpick but wouldn't ".. never present in version 1 of the repository format" be better wording? (Not sure why there would be a capital R in "repository" either, it's not in the earlier text).